packages/contentful--app-scripts/lib/clean-up/clean-up-bundles.js

```diff
@@ -29,20 +32,62 @@ const scheduleBundleDeletion = async (bundlesToDelete, client, settings) => {
 };
 
 async function cleanUpBundles(settings) {
```

Comment:
I wonder if it's possible to break up this function a little bit as it's really big now and includes few other functions in it.

Reply:
I can of course, though I'd rather clean up the whole thing and rewrite it to TS with proper modules.